flow/layers/backdrop_filter_layer.cc

```diff
@@ -62,7 +62,7 @@ void BackdropFilterLayer::Paint(PaintContext& context) const {
   paint.setBlendMode(blend_mode_);
   Layer::AutoSaveLayer save = Layer::AutoSaveLayer::Create(
       context,
-      SkCanvas::SaveLayerRec{&paint_bounds(), &paint, filter_.get(), 0});
+      SkCanvas::SaveLayerRec{&paint_bounds(), &paint, filter_.get(), 0}, true);
```

Comment:
Consider using an enum rather than a boolean so that the meaning is more obvious here, e.g. `Layer::AutoSaveMode::kLeafCanvas` or something.

Reply:
Alternatively, instead of providing the `PaintContext`, provide the `SkCanvas` and the flag about whether to checkerboard offscreen layers directly - although again perhaps with an enum rather than a bool to make the callsite more readable.

Reply:
I like the enum approach, updated the code to use a enum.
One thing is that SkCanvas has deleted the "=" operator, so I can't really assign the `canvas_` using "="
So something like:
```
switch (save_mode) {
    case blahblah:
    canvas_ = *internal_nodes_canvas;
    break;
    case blahblahblah:
    canvas_ = *leaf_nodes_canvas;
    break;
}
```
So I still kept the "if-else" clause in the constructor.

Reply:
Ahh ok. I guess we could wrap it in an sk_sp or something, but that's fine. I was assuming incorrectly that it already was an SkRefCnt.